README.md

```diff
@@ -23,6 +23,87 @@ gem install optimizely-sdk
 To access the Feature Management configuration in the Optimizely dashboard, please contact your Optimizely account executive.
 
 ### Using the SDK
+
+To use the SDK, the Optimizely instance can be initialized in three different ways as per your requirement. Ruby SDK also includes a factory class, OptimizelyFactory, which provides methods to create Optimizely instance with default configuration.
+
+ 1. Initialize Optimizely with a datafile. This datafile will be used as ProjectConfig throughout the life of Optimizely instance.
+    ~~~~~~~~~~~~
+    optimizely_instance = Optimizely::Project.new(datafile)
+2. Initialize Optimizely by providing an 'sdk_key'. This will initialize an HTTPConfigManager that makes an HTTP GET request to the URL ( formed using your provided sdk key and the default datafile CDN url template) to asynchronously download the project datafile at regular intervals and update ProjectConfig when a new datafile is recieved. 
+      ~~~~~~~~~~~~
+      optimizely_instance = Optimizely::OptimizelyFactory.create_default_instance_with_sdk_key('put_your_sdk_key_here')
+
+  The OptimizelyFactory class also exposes method to initialize Optimizely instance with sdk_key along with a default datafile. This hard-coded datafile will be used initially before any update.
+    
+    
+    optimizely_instance = Optimizely::OptimizelyFactory.create_default_instance_with_sdk_key_and_datafile('put_your_sdk_key_here', datafile)
+   
+ 3. Initialize Optimizely by providing a Config Manager that implements a 'get_config' method.You may use our HTTP Config Manager and customize it to your need. 
```

Comment:
Need a space after the first sentence. Suggest revising the second sentence to:

You can customize our HTTP Config Manager as needed.

Reply:
Also, does this mean the HTTPConfigManager described in the section just below? IF so, I suggest saying `HTTPConfigManager` instead.